pkg/controllers/context/context_test.go

```diff
@@ -34,49 +36,63 @@ func TestContext_IsControllerEnabled(t *testing.T) {
 			name:                         "on by name",
 			controllerName:               "bravo",
 			disabledByDefaultControllers: []string{"delta", "echo"},
-			controllers:                  []string{"alpha", "bravo", "-charlie"}, // --controllers=alpha,bravo,-charlie
```

Comment:
The comments help us to understand the test better and can be kept.

Reply:
@XiShanYongYe-Chang Made the suggested changes. Thanks!